### ISSUE TITLE
[User flows and navigating] There are no implemented entry points, or a recommended entry point for your new content or feature is missing.  (03.06.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#30430]
#### VFS team name
Public Websites

#### VFS product name
ORDMI Harassment Reporting Tool - Landing Page

#### Point of Contact/Reviewers
Mikki Northuis (IA)

---

### Platform Issue
There are no implemented entry points, or a recommended entry point for your new content or feature is missing. 

### Issue Details
Assets provided in staging review ticket do not show implementation of entry points - footer and Contact us page.  

### VA.gov Experience Standard
[Cateogy Number 03, Issue Number 06](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.5_AA

---

### Platform Recommendation
Implement correct entry points. 

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by brianalloyd** [comment]
@mnorthuis footer ticket is [here ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/29605)and will be implemented post launch of page.

**Comment by mnorthuis** [comment]
Thanks @brianalloyd!  I will validate and close this once that is live.  This is not launch-blocking.

**Comment by mnorthuis** [comment]
Here's the correct ticket for the addition to the footer: https://github.com/department-of-veterans-affairs/va.gov-team/issues/29606

**Comment by laflannery** [comment]
Following up on this older accessibility ticket since this page is now live, there is now a link to this in the footer, are we able to close this issue now?
